src/checkout/CheckoutForm.tsx

```diff
@@ -79,7 +78,7 @@ const CheckoutForm: FC = () => {
           >
             Upgrade to Usage-Based Account
           </h1>
-          {isFlagEnabled('credit250Experiment') && !isPaygCreditActive && (
+          {isFlagEnabled('credit250Experiment') && (
```

Comment:
Redundant because the `<GoogleOptimizeExperiment>` component will check if payg credit is active or not.